dll/win32/shell32/wine/shellord.c

```diff
@@ -743,7 +743,7 @@ void WINAPI SHAddToRecentDocs (UINT uFlags,LPCVOID pv)
 	if (SUCCEEDED(SHGetSpecialFolderLocation(hwnd, CSIDL_RECENT,
 						 &pidl))) {
 	    SHGetPathFromIDListA(pidl, link_dir);
-	    IMalloc_Free(ppM, pidl);
+	    IMalloc_Free(ppM,pidl);
```

Comment:
Unwanted change?

Reply:
Yes, I did not notice that the space is removed.

Reply:
Yes please.

Reply:
Done :)